src/target/riscv/riscv-013.c

```diff
@@ -977,27 +977,51 @@ static int execute_abstract_command(struct target *target, uint32_t command,
 	return ERROR_OK;
 }
 
-static riscv_reg_t read_abstract_arg(struct target *target, unsigned index,
-		unsigned size_bits)
+static void add_data_regs_read(struct riscv_batch *batch, unsigned int index,
```

Comment:
I've tried to find a more descriptive name:

```suggestion
static void abstract_data_read_fill_batch(struct riscv_batch *batch, unsigned int index,
```

Reply:
Addressed.